### ISSUE TITLE
Can't copy from Web Developer tools

### ISSUE THREAD

**Issue by ArtekMisiek** [zen-browser/desktop#1862]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

I can't copy anything with CMD + C (on mac) or CTRL + C (on windows) from developer console. I tried copying from:
- CSS rules
- Console

In network tab when i want to copy request response it copies the request itself.

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.1-a.7

### What platform are you seeing the problem on?

macOS - aarch64, Windows

**Comment by chriskirknielsen** [comment]
Seems like a regression (#1563), I can confirm I also cannot copy from DevTools.

One detail, though: if I click a property name, value, or selector in the CSS rules panel so as to trigger its "edit mode", I can actually copy from that "input". However, selecting arbitrary text like multiple lines in the CSS rules, does not get copied with `Cmd+C`.

TL;DR: Edit mode: copy works. Read mode selection: copy doesn't work.

**Comment by ArtekMisiek** [comment]
Is this getting picked up for fixing?

**Comment by AyushPatel2208** [comment]
Can't copy from the Developer console. We have to explicitly right click and copy the required variable.

**Comment by ArtekMisiek** [comment]
It's fixed
